metals/src/main/scala/scala/meta/internal/worksheets/DecorationWorksheetPublisher.scala

```diff
@@ -40,7 +43,10 @@ class DecorationWorksheetPublisher() extends WorksheetPublisher {
       .map { s =>
         new DecorationOptions(
           s.position().toLsp,
-          new MarkedString("scala", s.details()),
```

Comment:
@deprecated use MarkupContent instead. from the LSP docs.

https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_hover

Reply:
Do we need to use MarkupContent though? MarkedString is deprecated for Hover, but will it be removed altogether?

Reply:
Talked about his offline and we are good to move forward 👍